grpc.go

```diff
@@ -123,6 +167,27 @@ func onlineResources(resource onlineResource, nbResources int32) (uint32, error)
 	return count, nil
 }
 
+func offlineCPUResources(nbCpus uint32) error {
+	resource := onlineResource{
+		sysfsOnlinePath: sysfsCPUOnlinePath,
+		regexpPattern:   cpuRegexpPattern,
+	}
+
+	var count uint32
+	for i := uint32(0); i < onlineCPUMaxTries; i++ {
+		r, err := offlineResources(resource, int32(nbCpus-count))
+		if err != nil {
+			return err
+		}
+		count += r
+		if count == nbCpus {
+			return nil
+		}
+		time.Sleep(onlineCPUMemWaitTime)
```

Comment:
Again, the name is slightly confusing to me. But also, doesn't this assume that offlining takes the same amount of time as onlining?

Reply:
Yes, I assume that.
But I can replace it with changeCPUMemStateWaitTime if you have concern